File: Prism.Scripting/src/Prism/Renderer/MeshFactory.cs

using System;
using System.Runtime.CompilerServices;

namespace Prism
{
    public static class MeshFactory
    {

        public static Mesh CreatePlane(float width, float height)
        {
            return new Mesh(CreatePlane_Native(width, height));
        }

        public unsafe static IntPtr CreatePlane_Native(float width, float height)
        {
            return InternalCalls.Prism_MeshFactory_CreatePlane(width, height);
        }

    }
}

File: PrismEditor/Assets/scripts/Python/Prism/Renderer/MeshFactory.py
import PrismNative as _Prism
from Prism.Renderer.Mesh import Mesh


class MeshFactory:
    @staticmethod
    def CreatePlane(width: float, height: float) -> "Mesh":
        handle = _Prism.Prism_MeshFactory_CreatePlane(width, height)
        return Mesh(handle)
